hypha/apply/users/views.py

```diff
@@ -159,7 +159,10 @@ class EmailChangeDoneView(TemplateView):
 
 @login_required()
 def become(request):
-    if not request.user.is_apply_staff:
```

Comment:
I believe this was an error, the form only shows for super admins so staff should not be allowed to use this function. Earlier staff did have access to this function.